raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -21,19 +21,25 @@ library ChannelManagerLibrary {
     /// @notice getChannelsForNode(address) to get channels that 
     /// the address participates in.
     /// @dev Get channels where the given address participates.
-    /// @param nodeAddress (address) the address of the node
-    /// @return (address[]) of the channel's addresses that nodeAddress participates.
-    function getChannelsForNode(Data storage self, address nodeAddress) constant returns (address[]) {
-        return self.nodeChannels[nodeAddress];
+    /// @param node_address (address) the address of the node
+    /// @return (address[]) of the channel's addresses that node_address participates.
+    function getChannelsForNode(Data storage self, address node_address)
+        constant
```

Comment:
So you'd prefer `constant returns (address[])` on one line here instead of splitting it up? Just to be sure that I understand what you mean.